services/conda/conda-base.js

```diff
@@ -5,6 +5,7 @@ const { nonNegativeInteger } = require('../validators')
 const { BaseJsonService } = require('..')
 
 const condaSchema = Joi.object({
+  license: Joi.string(),
```

Comment:
We should either make `license` a `.required()` key, or if it is optional in the response then `CondaLicense` should have its own schema that only requires `license`.

Reply:
Some packages have the license as null. For example [this](https://api.anaconda.org/package/conda-forge/python). I'm not secure if license key are included in all packages, neither if the schema match with null licenses. I'll take a look at it again.

Reply:
OK. If license may be null or missing, then we don't want to make it required to render the platform/version badges. Lets define a custom schema in `conda-license.service.js` for just the license:

```js
const schema = Joi.object({
  license: Joi.string().required(),
}).required()
```

and make it required for the license badge only.

Reply:
I forget offhand whether a `null` value would fail that schema validation, but as long as we can throw an error and present a helpful message (like no license found) as opposed to the `invalid response data` that'd be good